demo/docker/docker-compose-env.yml

```diff
@@ -97,6 +97,7 @@ services:
     volumes:
       - ./flink/flink-conf.yaml:/tmp/flink-conf.yaml
       - ./flink/flink-entrypoint.sh:/flink-entrypoint.sh
+      - ./flink/postgresql-42.2.19.jar:/opt/flink/lib/postgresql-42.2.19.jar
```

Comment:
Is driver included? If not - we should probably extend quickstart description?
let's add some comment why it's here, and that it can be removed if one doesn't plan to use postgres enricher.
Also - let's mention that we don't link driver in nk, because it's included :)